dndtroops-api/src/main/java/cz/muni/fi/pa165/dndtroops/dto/CreateAdminDTO.java

```diff
@@ -0,0 +1,40 @@
+package cz.muni.fi.pa165.dndtroops.dto;
+
+import java.util.Objects;
+
+/**
+ * @author Miroslav Macor
+ */
+
+public class CreateAdminDTO {
+    private Long id;
```

Comment:
When you need to create an admin entity with a specific name, you don't know his database ID beforehand and this field will always be null. Hence, field `id` in CreateAdminDTO is not needed.